frontEnd/src/main.jsx

```diff
@@ -3,8 +3,19 @@ import ReactDOM from 'react-dom/client';
 import App from './App';
 import './index.css';
 import { BrowserRouter } from 'react-router-dom';
+import { Auth0Provider } from '@auth0/auth0-react';
+
+
 ReactDOM.createRoot(document.getElementById('root')).render(
-    <BrowserRouter>
-        <App />
-    </BrowserRouter>
+    <Auth0Provider
+        domain="hacker-camp.jp.auth0.com"
+        clientId="x2QfwLOOV9I4s4f62PuL1OSzzN3b5Pdp"
```

Comment:
It is a good practice to maintain configuration in environment variables to handle different types(local, dev, prod) of deployments